docs/troubleshooting/uninstall.md

```diff
@@ -0,0 +1,21 @@
+# OSM Uninstall Troubleshooting Guide
+
```

Comment:
Can we additionally list how to uninstall here before jumping into leaked resources directly? If there is an uninstall guide, I'd refer to it here to mention something along the lines of `"If the uninstall guide xyz was not followed, resources could be leaked..."`

Reply:
Created uninstall doc and linked to it in the troubleshooting doc